spec/kitchen_spec/directions/bake_figure_spec.rb

```diff
@@ -230,6 +230,14 @@
         end
       end
     end
+
+    context 'when link to figure has class' do
+      it 'stores link with class' do
+        pantry = book1.pantry(name: :link_type)
+        expect(pantry).to receive(:store).with('figure-label', { label: 'someId' })
```

Comment:
I noticed that the classes hasn't been changed in spec files, but I leave it up to you, if you think if it's worth change it also here to make the tests more readable. :slightly_smiling_face: